pom.xml

```diff
@@ -35,6 +35,7 @@
   </scm>
 
   <properties>
+    <aws-java-sdk.version>1.12.276</aws-java-sdk.version>
```

Comment:
lets use this variable here as well - https://github.com/ExpediaGroup/beekeeper/blob/main/beekeeper-cleanup/pom.xml#L14

Reply:
use here - https://github.com/ExpediaGroup/beekeeper/blob/main/beekeeper-cleanup/pom.xml#L86

remove from here - https://github.com/ExpediaGroup/beekeeper/blob/main/beekeeper-cleanup/pom.xml#L14

Reply:
updated, also change version to `1.12.311` to match it with previously version